pages/axelar-vm/overview.mdx

```diff
@@ -0,0 +1,88 @@
+* **Developers** - Think Interchain and more easily take your dApps further
+* **Integrators** - Connect your blockchain or tech stack to the Axelar Network
+* **Users** - Improved abstractions will make using web3 technologies simpler, safer, and easier
+
+## Foundation
+The Foundation of the Axelar VM is the ability to deploy and run smart contracts written in [CosmWasm](https://book.cosmwasm.com/). 
+
+<Callout emoji="💡">
+  Developers interested in trying out smart contract support on the Axelar Blockchain can give it a try on the [Axelar Wasm Devnet](https://bright-ambert-2bd.notion.site/Axelar-Devnet-885a966bf4764151a7e4c1fde68a04a2).
+</Callout>
+
+dApps built using CosmWasm are executed by the Axelar Network's [validators](https://axelarscan.io/validators), and are part of the state of the blockchain, creating an application layer on top of the existing consensus layer. Enterprising developers can use this application layer to begin to write connections to external infrastructure.
+
+## Interchain Amplifier
+The magic of this foundation of smart contracts manifests most strongly when integrators are able to plug new connections directly into the Axelar Network, allowing new paths, new chains, and more configurability for cross chain communication.
+
+We're designing paths for any integrator to create modules (contracts) deployed on Axelar that act as gateways to other ecosystems.
+
+### How Axelar previously integrated with blockchains
+To understand the future of our Interchain Amplifier, it can be helpful to understand how Axelar got to where we are today.
+
+Axelar has 30+ blockchain integrations that allow a combination of [message passing](/dev/general-message-passing/overview) and [token transfers](/dev/send-tokens/overview). Each of these blockchains requires implementation of technologies from several different parts of the stack. Today, Axelar validators execute code as part of the Axelar Node Software's consensus layer, requiring a full network upgrade (including downtime) for every modification or addition of a new chain.
+
+### Integrating a Target Chain with the Interchain Amplifier
+There are several required components that you must build to integrate with the Interchain Amplifier. These components are spread across the Axelar Blockchain, the integration target, and code executed by validators.
+
+1. **Target Chain Gateway** - On each chain connected to Axelar, a gateway is deployed. This gateway is capable of receiving incoming messages and token transfers. The gateway natively knows how to lock, burn, mint, and release tokens as appropriate, and how to emit events that can be consumed externally. This gateway has knowledge of the Axelar validator set and is capable of receiving incoming approvals and verifying the signatures from those validators. This gateway must be built in the native language of the integration target (eg Solidity for EVM Chains).
+1. **Validator Companion Process**
```

Comment:
This process doesn't need to be run by the mainnet validators. This will be run by anyone who wishes to verify transactions have occurred on a particular chain. Each chain will have their own set of staked entities verifying transactions, and this set is independent of the Axelar validators.